runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowStreamingPipelineOptions.java

```diff
@@ -211,6 +211,14 @@ public interface DataflowStreamingPipelineOptions extends PipelineOptions {
 
   void setWindmillServiceStreamMaxBackoffMillis(int value);
 
+  @Description(
+      "If true, Dataflow streaming pipeline will be running in direct path mode."
+          + " VMs must have IPv6 enabled for this to work.")
+  @Default.Boolean(false)
+  boolean getIsWindmillServiceDirectPathEnabled();
+
+  void setIsWindmillServiceDirectPathEnabled(boolean isWindmillServiceDirectPathEnabled);
```

Comment:
I would remove the `is`, otherwise specifying the flag looks odd IMO
`--isWindmillServiceDirectPathEnabled=true`